drivers/bridge/bridge.go

```diff
@@ -147,6 +147,27 @@ func (c *networkConfiguration) Validate() error {
 	return nil
 }
 
+func (c *networkConfiguration) copyDefaultConfigs(option options.Generic, config *networkConfiguration) {
```

Comment:
We cannot differentiate if the config object is set or unset by the user. The only way to find out if the user has an explicit configuration is by looking at the options.

I will change the actors.